caluma/extensions/events/work_item.py

```diff
@@ -169,16 +169,7 @@ def send_rejection_mail(sender, work_item, user, **kwargs):
     _send_work_item_mail(work_item)
 
 
-@on(post_create_work_item, raise_exception=True)
-@on(post_reopen_case, raise_exception=True)
-@transaction.atomic
-def set_case_status_post_create_work_item_reopen_case(
-    sender,
-    user,
-    work_item=None,
-    case=None,
-    **kwargs,
-):
+def set_case_status(case=None, work_item=None):
```

Comment:
This function has so little common code, I'd just split it in two and get rid of the whole `if workitem else` stuff

Reply:
Agreed and done :+1:

Reply:
Agreed and done :+1:

Reply:
This messes up my precious coverage 😖